### ISSUE TITLE
Validate permissions against ticket #372

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#797]
This ticket lays out how permissions should work for multi-shop so we need to double-check these have not changed.

**Comment by aaronjudd** [comment]
from @newsiberian 
> Hello, could you please check "core" permissions. Looks like they missing inside accounts "Edit member" settings. Thats because of they failed on if (pkg.registry && pkg.enabled) { check [here](https://github.com/reactioncommerce/reaction/blob/development/packages/reaction-accounts/client/templates/members/member.js#L61)

First glance, packages.enabled is not needed here..

**Comment by Capt-Slow** [comment]
@zenweasel It appears I am blocked by #799. I am unable to create a second shop, and successfully navigate to that second shop. I was made aware of 'Meteor.call("shop/createShop")' which would potentially force a second shop creation. However, doing so would not be a valid test case.

**Comment by zenweasel** [comment]
@Capt-Slow How are you blocked from writing tests for this because this UI doesn't support it?

**Comment by Capt-Slow** [comment]
@zenweasel It doesn't appear that I am able to successfully create a second shop (even though there is data placeholders in Mongo). I attempted to create a second shop by renaming '<blank_site>' in the DB. Now if I force a second shop creation using the Meteor.call, I was told it's kind of a hack. So it appears the feature of multi-shops isn't complete. The tests I write in which case should fail. From my limited understanding, CI builds would also fail?

**Comment by zenweasel** [comment]
Ok, can we create a ticket or a test-plan for what tests for a complete implementation of multi-shops would look like? Starting with being able to programmatically create a shop I would think.

**Comment by zenweasel** [comment]
Closing in favor of #2075
